src/oceans/components/common/Guide.jsx

```diff
@@ -41,11 +111,21 @@ let UnwrappedGuide = class Guide extends React.Component {
     }
 
     // Start playing the typing sounds.
-    if (!state.guideShowing && !state.guideTypingTimer && currentGuide) {
+    if (
+      !state.textToSpeechLocale &&
```

Comment:
Probably fine, but we'll be turning off typing sounds even if we aren't able to successfully play TTS for a given locale (if one is provided), yeah?

Reply:
Yes.  It's definitely worth keeping an eye on whether there are situations where we'd prefer the sounds to be left on.